types/underscore/index.d.ts

```diff
@@ -5306,13 +5306,13 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.sortBy
         **/
-        sortBy(iterator?: _.ListIterator<T, any>, context?: any): _Chain<T>;
+        sortBy<TSort>(iterator?: _.ListIterator<T, TSort>, context?: any): _Chain<T, T[]>;
```

Comment:
Is the new `TSort` type parameter really necessary, and if so, what for? I wonder whether the `_Chain<T, T[]>` return type wouldn't already be a sufficient solution by itself.

Reply:
You're right as far as I'm able to see - I cargo-culted it from the other `sortBy` implementation further up. I'll change that back. Thanks!

Reply:
My codebase is happy with it. :+1: